src/plugins/DistanceMeasurementsPlugin/index.js

```diff
@@ -1,4 +1,61 @@
 export * from "./DistanceMeasurementsPlugin.js";
 export * from "./DistanceMeasurementsControl.js";
 export * from "./DistanceMeasurementsMouseControl.js";
-export * from "./DistanceMeasurementsTouchControl.js";
\ No newline at end of file
+export * from "./DistanceMeasurementsTouchControl.js";
+
+import {Component} from "../../viewer/scene/Component.js";
+import {activateDraggableDots} from "../../../src/plugins/lib/ui/index.js";
+
+class DistanceMeasurementEditControl extends Component {
+
+    /**
+     * Edits {@link DistanceMeasurement} with mouse and/or touch input.
+     *
+     * @param {DistanceMeasurement} [measurement] The DistanceMeasurement to edit.
+     * @param [cfg] Configuration
+     * @param {PointerLens} [cfg.pointerLens] A PointerLens to use to provide a magnified view of the cursor.
+     * @param {boolean} [cfg.snapping] Whether to enable snap-to-vertex and snap-to-edge.
+     * @param {boolean} [handleMouseEvents] Whether to hangle mouse input.
+     * @param {boolean} [handleTouchEvents] Whether to hangle touch input.
+     */
+    constructor(measurement, cfg, handleMouseEvents, handleTouchEvents) {
```

Comment:
Could we please put the handleMouseEvents and handleTouchEvents into cfg

Reply:
Please note the `DistanceMeasurementEditControl` is a "private" class, so its arguments are not at the moment part of the public API.
The public classes are `DistanceMeasurementEditMouseControl` and `DistanceMeasurementEditTouchControl`.
If `handleMouseEvents` and `handleTouchEvents` were to be properties of the `cfg` object, how should these properties be handled if already set inside the public classes' `cfg` argument.

Reply:
Right, keeping handleMouseEvents and handleTouchEvents separate makes sense then